src/Molecules/Editable/Editable.stories.jsx

```diff
@@ -40,14 +38,14 @@ const CustomFormControl = ({
     <InputLabel>
       Label
       <FlexBox gap={1}>
-        <Select value={first} onChange={e => onChange([e.target.value, second])}>
+        <Select value={first} onChange={e => onChange([e.target.value, second])} native>
           {options.map(({ value, label }) =>
-            <option value={value}>{label}</option>,
+            <Option key={value} value={value}>{label}</Option>,
```

Comment:
Why did you need to add a `key`?

Reply:
https://reactjs.org/docs/lists-and-keys.html#keys